src/clients/python/library/grpcclient.py

```diff
@@ -893,6 +894,11 @@ def infer(self,
             model-specific action such as terminating the request. If not
             provided, the server will handle the request using default setting
             for the model.
+        client_timeout : int
```

Comment:
How is this timeout indicated to the client? A special exception?

Reply:
They will see an InferenceServerException with message "Deadline Exceeded" message.